lib/DUAL_MIT.tpl

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013-2015 StrongLoop, Inc.
+Copyright (c) 2013-2015 StrongLoop, Inc<%= others %>.
```

Comment:
Look again, I think it would only cause a spelling mistake if I did what you suggest in https://github.com/strongloop/strong-tools/pull/32#discussion-diff-40500505, as-is, it means that the copyright line will always end in a `.`.